line.go

```diff
@@ -0,0 +1,65 @@
+package xinvoice
+
+import "github.com/invopop/gobl/bill"
+
+// Line defines the structure of the IncludedSupplyChainTradeLineItem in the CII standard
+type Line struct {
+	ID              string           `xml:"ram:AssociatedDocumentLineDocument>ram:LineID"`
+	Name            string           `xml:"ram:SpecifiedTradeProduct>ram:Name"`
+	NetPrice        string           `xml:"ram:SpecifiedLineTradeAgreement>ram:NetPriceProductTradePrice>ram:ChargeAmount"`
+	TradeDelivery   *Quantity        `xml:"ram:SpecifiedLineTradeDelivery>ram:BilledQuantity"`
+	TradeSettlement *TradeSettlement `xml:"ram:SpecifiedLineTradeSettlement"`
+}
+
+// Quantity defines the structure of the quantity with its attributes for the CII standard
+type Quantity struct {
+	Amount   string `xml:",chardata"`
+	UnitCode string `xml:"unitCode,attr"`
+}
+
+// TradeSettlement defines the structure of the SpecifiedLineTradeSettlement of the CII standard
+type TradeSettlement struct {
+	TaxType        string `xml:"ram:ApplicableTradeTax>ram:TypeCode"`
+	TaxCode        string `xml:"ram:ApplicableTradeTax>ram:CategoryCode"`
+	TaxRatePercent string `xml:"ram:ApplicableTradeTax>ram:RateApplicablePercent"`
+	Sum            string `xml:"ram:SpecifiedTradeSettlementLineMonetarySummation>ram:LineTotalAmount"`
+}
+
+func newLine(line *bill.Line) *Line {
+	return &Line{
+		ID:       line.Item.Name,
+		Name:     line.Item.Name,
+		NetPrice: line.Item.Price.String(),
+		TradeDelivery: &Quantity{
+			Amount:   line.Quantity.String(),
+			UnitCode: string(line.Item.Unit.UNECE()),
+		},
+		TradeSettlement: &TradeSettlement{
```

Comment:
I'm guessing this is just a first version, but I'd also split this out into a constructor. It probably also needs to support an array in case there are multiple taxes on the invoice line.